bin/test-workflow/test_app_summon/Dockerfile.builder

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.0
+FROM ruby:3.1-slim-bullseye
```

Comment:
Is the `slim` image going to be enough for this?

Reply:
Come to think of it, is this even used anywhere?

Reply:
First step was to build and get obvious failures to see if slim would be enough. But, no, I don't see it used anywhere... Looks like the main Dockerfile makes a builder now instead of using the Dockerfile.builder. I'll try removing it.

Reply:
Do you think we can just remove the whole 'test_app_summon' folder?